src/plugins/intel_gpu/include/intel_gpu/primitives/arg_max_min.hpp

```diff
@@ -43,6 +43,42 @@ struct arg_max_min : public primitive_base<arg_max_min> {
           sort(sort),
           values_first(values_first) {}
 
+    /// @brief Constructs arg_max_min for top_k constant
+    arg_max_min(const primitive_id& id,
+                const input_info& input,
+                ov::op::TopKMode mode,
+                uint32_t top_k,
+                int64_t axis,
+                ov::op::TopKSortType sort = ov::op::TopKSortType::SORT_VALUES,
+                bool values_first = false,
+                const padding& output_padding = padding(),
+                data_types output_data_type = data_types::f32,
+                const size_t num_outputs = 1)
+        : primitive_base(id, {input}, {output_padding}, {optional_data_type{output_data_type}}, num_outputs),
+          mode(mode),
+          top_k(top_k),
+          axis(axis),
+          sort(sort),
+          values_first(values_first) {}
+
+    /// @brief Constructs arg_max_min for top_k parameter
+    arg_max_min(const primitive_id& id,
```

Comment:
BTW can't we use this as the first API?

Reply:
I think we can set top_k value in creation as 0 if the 2nd node is not constant.

Reply:
done